bin/install-wordpress.sh

```diff
@@ -65,6 +65,7 @@ fi
 # Make sure the uploads and upgrade folders exist and we have permissions to add files.
 echo -e $(status_message "Ensuring that files can be uploaded...")
 docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm $CONTAINER chmod 767 /var/www/html/wp-content/plugins
+docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm $CONTAINER chmod 767 /var/www/html/wp-config.php
```

Comment:
Multiple file paths can be passed to a single `chmod` command. Could (marginally) improve performance to avoid multiple separate `docekr-compose run`.

Reply:
I'm wondering if we still need it to be honest. I suspect that we were using the wrong user for WP-CLI dedicated to e2e tests. However, I wanted to unblock other folks and I didn't investigate further.